packages/node/src/evm/requests/api-calls.ts

```diff
@@ -80,12 +81,17 @@ export function applyParameters(request: ClientRequest<ApiCall>): LogsData<Clien
   return [[], { ...request, parameters }];
 }
 
+export interface UpdatedFulfilledRequests {
+  logs: PendingLog[];
+  requests: ClientRequest<ApiCall>[];
+}
+
 export function updateFulfilledRequests(
   apiCalls: ClientRequest<ApiCall>[],
   fulfilledRequestIds: string[]
 ): LogsData<ClientRequest<ApiCall>[]> {
   const { logs, requests } = apiCalls.reduce(
-    (acc, apiCall) => {
+    (acc: UpdatedFulfilledRequests, apiCall) => {
```

Comment:
Again, I slightly prefer when the initial value is typed instead (you also get a bit more type checking that way).